site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -59,7 +59,22 @@ The folder structure is based on the [Cypress scaffold](https://docs.cypress.io/
    - If element ID is not present in the webapp, you may add it in `camelCase` form with human readable name (e.g. `<div id='sidebarTitle'>`). Watch out for potential breaking changes in the snapshot of the unit testing.  Run `make test` to see if all are passing, and run `npm run updatesnapshot` or `npm run test -- -u` if necessary to update snapshot testing.
 2. Add commands or shortcuts to `cypress/support/commands.js` (e.g. `toAccountSettingsModal`) that makes it easier to access a page, section, modal and etc. by simply using it as `cy.toAccountSettingsModal('user-1')`.
 3. Organize `cypress/integration` with a subfolder to group similar tests.
-4. Refer to [this pull request](https://github.com/mattermost/mattermost-webapp/pull/2058/files#diff-c42a18e742b351c0ade058ed0c4b5c5eR10) as a guide on how to write and submit an end-to-end testing PR.
+4. Add test key and description in the spec file.  This is used to track test cases in our Release Testing spreadsheet, usually attached to our [Release Candidate Testing](https://community.mattermost.com/core/pl/deocy9tx9pfbbjka9prwie6u9e) announcement.
```

Comment:
Would it make sense to link to the channel instead of a past post, such as:

`This is used to track test cases in our core staff Release Testing spreadsheet, which is linked in the header of the [Release Discussion channel](https://community.mattermost.com/core/channels/release-discussion) during release testing.`

Also wonder if it's worth mentioning "core staff" to help avoid others going through the confusion of clicking it, requesting access, then not getting access (it's a private doc)?

Reply:
Thanks, will update that.
re: private doc, I thought it's public since I could access it using my personal email.  But I agree that it makes sense to clarify it as core staff.

Reply:
Now updated.  Please take a look again, thanks @lindalumitchell!

![screen shot 2019-02-12 at 1 31 37 am](https://user-images.githubusercontent.com/5334504/52581623-268fa200-2e66-11e9-9f2c-9ec906f1ab04.png)